src/CryptoAxus.Application/Dto/BiddingDto.cs

```diff
@@ -0,0 +1,20 @@
+﻿namespace CryptoAxus.Application.Dto;
+
+public class BiddingDto : BaseDto
+{
+    public string? NftId { get; set; }
+
+    public decimal? BiddingPrice { get; set; }
+
+    public string? ArtistWalletAddress { get; set; }
+
+    public string? FromWalletAddress { get; set; }
+
+    public string? BidderUserName { get; set; }
+
+    public bool? HighestBid { get; set; }
+
+    public BiddingDto()
+    {
+    }
+}
```

Comment:
Add another constructor that takes parameters for each property and assigns values to it.

Reply:
Resolved